distribution/docs/src/main/resources/content/_configuring/hardening-solr-index.adoc

```diff
@@ -10,25 +10,12 @@
 * *{hardening-step}*
 
 The ${branding}'s design includes support for pluggable indexes.
-The default installation contains a Solr index to be used as the Metadata Catalog, but this implementation can be replaced with an alternate 3rd party index implementation.
+The default installation contains a Solr index to be used as the Metacard Catalog, but this implementation can be replaced with an alternate 3rd party index implementation.
 
 The following sections provide hardening guidance for Solr; however, they are to serve only as reference and other an additional security requirements may be added.
 
-The Solr Admin user interface uses basic authentication as part of the server configuration.
-
-=== Configuring Solr Node Security
-
-The Solr server is protected by the built in REST security architecture.
-The configuration can be changed by editing the <<_configuring_the_web_context_policy_manager,Web Context Policy Manager>>
- configuration for the `/solr` web context.
-
-. Navigate to the *${admin-console}*.
-. Select the *${ddf-security}* application.
-. Select the *Configuration* tab.
-. Select *Web Context Policy Manager*.
-
-By default, the configuration is set to `/solr=SAML|PKI|BASIC`.
-This allows a user or another system to connect to Solr using any of those authentication methods.
+<<TODO: Internally manged should use 2-way autho and refuse remote.>>
+<<TOOD: Externally managed should use HTTPS>>
```

Comment:
Will this be filled out in this PR or after other tickets?

Also maybe this is a good example of how our docs tests could be improved... We probably shouldn't allow things to be marked as links that go nowhere.
![image](https://user-images.githubusercontent.com/11355332/41485819-e4d69f2c-7096-11e8-8e77-6de05ff2d892.png)